Makefile

```diff
@@ -0,0 +1,16 @@
+serve:
+	cd docs/html && python -m SimpleHTTPServer
```

Comment:
This will depend on the python version that is executed with `python`, should specify version.

Reply:
Rather than specify the version, the change I've made is to detect the version and launch the right module for the detected version. That way if you only happen to have one installed, we're not relying on it being either 2 or 3.

Reply:
What will happen if you call make serve twice?

There is no target to shut down the webserver and it will run forever.

For what do we need a webserver, anyway? One can directly point a webbrower onto the html directory (file://).

Reply:
@schaecsn
> For what do we need a webserver, anyway? One can directly point a webbrower onto the html directory (file://).

Up-to-date Linux distributions have started sandboxing their browsers making most `file://` paths nonfunctional. 

However, given the complexity of making this functionality cross-OS, cross-python, it seems more reasonable to simply leave this target out and leave serving the files to everyone on their own.